config/database.yml

```diff
@@ -2,8 +2,8 @@ default: &default
   adapter: postgresql
   encoding: unicode
   pool: 5
-  username: <%= ENV['DB_USERNAME'] || 'postgres' %>
-  password: <%= ENV['DB_PASSWORD'] || '' %>
+  username: <%= ENV['DB_USERNAME'] || 'manage_courses_backend' %>
```

Comment:
I'm going to check if we need to update the docker setup to handle this, but we probably do.

Reply:
Works fine. 👍